docs/benchmarks.md

```diff
@@ -164,9 +164,10 @@ before sharing these JSON files. Environment variables with names containing the
 
 ## Automating Benchmarks
 
-For convenience, the [benchmark.py](../integration_tests/src/main/python/benchmark.py) script is
-provided, allowing benchmarks to be run in an automated way with multiple configurations. Example
-usage is provided in the documentation within the script.
+For convenience, the
+[benchmark.py](https://github.com/NVIDIA/spark-rapids/blob/branch-0.3/integration_tests/src/main/python/benchmark.py)
```

Comment:
This link works just fine in the version at https://github.com/NVIDIA/spark-rapids/blob/branch-0.3/docs/benchmarks.md.  What's the issue with relative links?  Making these absolute links has a huge downside: they link to the _wrong version of the target_ when we move to the next release 😞 

We need to find a solution to make these links continue working as relative, or we're trading one problem for another.  There are a number of other relative links being used that are not updated here, so how are those working when this does not?

Reply:
When we push these pages to gh-pages, we will need the absolute link since this is a pointer to another part of our repository.  When we release 0.3 I intend to change the link to https://github.com/NVIDIA/spark-rapids/blob/main/integration_tests/src/main/python/benchmark.py so it will point to a stable location.  

I will leave it as relative within branch-0.3 so it continues to point to the current branch going forward.